source/geometries/Next100FieldCage.cc

```diff
@@ -42,18 +45,27 @@ Next100FieldCage::Next100FieldCage():
   GeometryBase(),
   // Dimensions
   active_diam_ (984 * mm), // distance between the centers of two opposite panels
-  gate_cathode_centre_dist_ (1205. * mm), // distance between gate and the centre of cathode grid
+  active_cathode_dist_ (1178.*mm), //distance from the  beginning of the teflon drift panels to the first side of the cathode.
```

Comment:
It's not clear to me what this distance represent. The beginning of the teflon drift is not the same as the beginning of the active region, right? Also, what is `the first side of the cathode`? In the past we used `beginning` and `end`, taking positive z as direction, for instance.

Reply:
It is actually the length of the teflon drift panels. If it is okay I think I should change the name of the variable to `teflon_drift_length_` to make the nomenclature clearer.

Reply:
Yes, that would be more straightforward. In the comment you can say `distance from the gate to the beginning of the cathode volume`

Reply:
ok!